### PR TITLE
Initial home page example.

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
- [x] Check the Layouts (because I changed those)
- [x] Check the homepage (because that's new

### CODE REVIEW

**Comment by joetower** [comment]
@ModulesUnraveled  Is this one ready for review? In working on #26 I noticed that the hero is off on [the homepage](http://localhost:6006/?path=/story/pages-landing-pages-home--home). I am seeing the following when comparing the twig version and this one:
heros-compared

File: 05-pages/landing-pages/landing-pages.stories.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import TextWithImage from '../../02-molecules/TextWithImage/TextWithImage.component';
+import Hr from '../../01-atoms/hr/Hr.component';
+import Grid from '../../03-organisms/grid/Grid.component';
+import NewsCard from '../../02-molecules/cards/NewsCard/NewsCard.component';
+import Cta from '../../02-molecules/cta/Cta.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Pages/Landing Pages/Home' };
```
Comment (joetower) [inline 05-pages/landing-pages/landing-pages.stories.js:17]: We can probably remove the `Home` here so `Home` isn't duplicated in the storybook sidebar nav. 
Screenshot-2020-06-16-09-52-57-210x194

**Review by joetower — CHANGES_REQUESTED** [review]

**Comment by joetower** [comment]
Also, I see in Jira that this might not actually be ready for review, so... yeah. Holler when this is ready and I can finish reviewing. :+1:

File: 05-pages/landing-pages/landing-pages.stories.js (hunk 1 above)
Comment (ModulesUnraveled) [reply 05-pages/landing-pages/landing-pages.stories.js:17]: done

**Review by joetower — APPROVED** [review]
@ModulesUnraveled :clap: :clap: :shipit:

**Pull request merged**